package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "eslint-config-wikimedia",
-	"version": "0.21.0",
+	"version": "0.22.0",
```

Comment:
This isn't a release commit; these are bumped in release commits only.

Reply:
Fixed